build/commands/scripts/commands.js

```diff
@@ -455,6 +455,8 @@ program
 
 program
   .command('run_perf_tests [perf_config] [targets]')
+  .option('--target_os <target_os>', 'target OS')
```

Comment:
do we also need target_environment here for ios? If ios is not supported we should have a console error/exit?

Reply:
we don't support ios perf tests yet.

Reply:
passing `--target_os ios` result in an error [here](https://github.com/brave/brave-core/blob/mplesa-run-perf-smoke-tests-label/tools/perf/components/common_options.py#L97)